### DISCUSSION TITLE
Are you changing https://hub.docker.com/r/timberio/vector like you're moving to apt.vector.dev and yum.vector.dev

### DISCUSSION THREAD

**Discussion by EdN-terascope** [vectordotdev/vector#19164] [General]
With the apt and yum/dnf repositories moving to apt.vector.dev and yum.vector.dev, are you also moving the DockerHub repository (https://hub.docker.com/r/timberio/vector) too?

**Comment by jszwedko** [comment]
We do plan to migrate it, but no updates yet. There will be an announcement when we start cutting over.
